app/Filament/Resources/UserResource.php

```diff
@@ -17,11 +17,6 @@ class UserResource extends Resource
 {
     protected static ?string $model = User::class;
 
-    public static function getLabel(): string
-    {
-        return trans_choice('strings.users', 1);
-    }
```

Comment:
Again here: Users is not panel related word!

Reply:
Yes, Users are a resource. Resources don't make sense to be translated.